FALCON/Source/FalconSimul.m

```diff
@@ -265,7 +265,7 @@
     elseif strcmp(estim.Reg, 'L1Smooth')
         Smoothed = (max(k(RegSmooth), [], 2) - min(k(RegSmooth), [], 2)) < 0.1;
         RP =(mean(k(RegSmooth), 2)) > 0.01;
-        Nparams = sum(Smoothed .* RP) + size(RegGroups, 2) * sum(~Smoothed .* RP);
+        Nparams = sum(Smoothed .* RP) + size(RegSmooth, 2) * sum(~Smoothed .* RP);
```

Comment:
that was probably a typo indeed, however in practice RegGroups and RegSmooth always were the same matrix (see driver lines 78/79)